src/app/[locale]/specs/repository/page.mdx

```diff
@@ -125,6 +125,38 @@ When importing CAR files, note that there may existing dangling CID references.
 
 The CARv1 specification is agnostic about the same block appearing multiple times in the same file ("Duplicate Blocks)". Implementations should be robust to both duplication and de-duplication of blocks, and should also ignore any unnecessary or unlinked blocks.
 
+## Repository Diffs
+
+A concept which supports efficient synchronization of data between independent services is "diffs" of repository trees between different revisions. The basic principle is that a repository diff contains all the data (commit object, MST nodes, and records) that have changed between an older revision and the current revision of a repo. The diff can be "applied" to the older mirror of the repository, and the result will be the complete MST tree at the current (newer) commit revision.
+
+Repo diffs can be serialized as CAR files, sometimes referred to as "CAR slices". Some details about diff CAR slices:
```

Comment:
Should we a. link to the CAR spec, b. specify that it's CAR v1?

Reply:
there is discussion elsewhere in this same spec file about CAR files generally (actually in the section just above this one). I didn't want to duplicate that level of granularity so I kind of just reference it. I think the desire is to have it be "the same" more than any specific format.